### DISCUSSION TITLE
noob question

### DISCUSSION THREAD

**Discussion by Rinus-Verdonschot-UB** [asreview/asreview#1868] [Q&A]
Sorry a noob question from my side..  I am making a guide to use ASReview for my university.
I have a very small dataset of which i labelled 8 items (5 priors and 3 after priors).

![afbeelding](https://github.com/user-attachments/assets/46fde43a-8871-453c-89b5-538eaa9e354a)

when I export to excel.. I see that all the articles have gotten a label (at included).

![afbeelding](https://github.com/user-attachments/assets/9b467329-2cc5-49b3-afeb-1cb9e745c3d7)

But.. according to your instructions.. the things i did not label should not have a label..

![afbeelding](https://github.com/user-attachments/assets/d368845b-02ae-43a7-bf30-617ce0413fac)

probably it is easy or i am confused or something.. but could you explain why I get labels (incl/excl) for things I did not label yet?

**Comment by Rensvandeschoot** [comment]
Could you share your project file with us so that we can take a look at what is going on and reply here? You can send it to asreview@uu.nl

**Reply by Rinus-Verdonschot-UB** [reply]
Dear Rens,

I have sent the project file and the excel output to you. Thanks for any suggestions you could give me.

Kind regards,

Rinus

**Reply by Rinus-Verdonschot-UB** [reply]
(After talking with Rens) I uninstalled and reinstalled asreview. The problem was solved then. No solution as to why it occurred but it does not occur anymore only reviewed items receive 0 or 1 at "included" column.
